src/commands/utils/modules.ts

```diff
@@ -45,6 +51,23 @@ class JahiaUtilsModule extends Command {
     const dependencies: string[] = flags.dependencies.split(',')
 
     const jahiaFullUrl = flags.jahiaUrl.slice(-1) === '/' ? flags.jahiaUrl : flags.jahiaUrl + '/'
+    for (let i = 0; i < flags.timeout; i++) {
+      let out = false
+      setTimeout(() => {
+        const response: any = new SyncRequestClient()
```

Comment:
I know we haven't always done it on this project, but I would really prefer to have the call in a different function